public/content/it/welcome/welcome-to-d.md

```diff
@@ -0,0 +1,58 @@
+# Benvenuto al tour di D
+
+Benvenuto al tour interattivo del *Linguaggio di Programmazione D*.
```

Comment:
I'm not sure about this. Switching to italian for lack of good translation.
Il complemento di stato in luogo si può esprimere sia con "a" che con "in" (in casi differenti) e non sono sicuro di quale sia corretto qui. Di solito diciamo cose tipo "al tour de France", "al gran tour". E' difficile trovare buoni esempi perché in effetti "tour" è una parola francese. "In" è solitamente approssimabile a "dentro" e "a" a "presso". A me personalmente "nel" piace di più in questo contesto, ma se hai informazioni più accurate su quale sia corretto, ben venga.
TL;DR Italian is a mess even for native speakers.